external/CapnProto.cmake

```diff
@@ -114,6 +122,7 @@ foreach (INCLUDE_DIR ${CAPNP_INCLUDE_DIRS})
   install(DIRECTORY ${INCLUDE_DIR}/capnp
```

Comment:
@scottpurdy It's confusing that these same INC and LIB installation steps are duplicated in src/CMakeLists.txt. Are they in fact accomplishing anything different and useful?